packages/block-editor/src/components/block-tools/style.scss

```diff
@@ -42,8 +34,8 @@
 
 	justify-content: center;
 
-	top: calc(50% - #{  $button-size-small * 0.5 });
-	left: calc(50% - #{  $button-size-small * 0.5 });
+	top: calc(50% - #{$button-size-small / 2});
```

Comment:
Apparently division like this has been deprecated in sass:

```
DEPRECATION WARNING
[1] : Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
[1] 
[1] Recommendation: math.div($button-size-small, 2)
```

So we can either use the `math.div` function, or just multiply by half as was used before. I will admit that the calc value overall is a bit obtuse here, so I wish there was a nicer way to solve this.